lib/workers/repository/process/limits.ts

```diff
@@ -48,7 +47,7 @@ export async function getConcurrentPrsRemaining(
     logger.debug(`Enforcing prConcurrentLimit (${config.prConcurrentLimit})`);
     let currentlyOpen = 0;
     for (const branch of branches) {
-      if (await branchExists(branch.branchName)) {
+      if (await platform.getBranchPr(branch.branchName)) {
```

Comment:
Basically, it reduces function result, because branches can exist without associated PRs.

Reply:
But as we don't count such branches, it means we potentially may create _more_ branches than before. Though, as we use same counter for now, corresponding PRs should be created too.